### ISSUE TITLE
Code Table Request - Zoo Local ID identifier

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#6453]
Initial Request

_**Goal**:  Create new identifier type = Zoo Local ID to replace the Albuquerque Biopark Local ID identifier currently in Arctos. 

_**Proposed Value**:  Zoo Local ID

_**Proposed Definition**:  The Zoo Local ID is the Local ID issued by zoos participating in the Species360 ZIMS database. Each zoo assigns a unique Local ID to animals in their care, and these values will change as the animal is transferred between zoos. It is important to record not only the Zoo Local ID but also the issuing agent to track which zoo assigned the ID.

__**Context**: The "Local ID" value is issued by each zoo participating in the ZIMS database. Each zoo in the network has a distinct Local ID in addition to a global GAN ID. By creating a "Zoo Local ID" type in Arctos, we can use this for all the different zoos participating in ZIMS, and use the "issued by" agent to specify which zoo issued the Local ID.  I propose using "Zoo Local ID" vs the "Local ID" type in ZIMS to more clearly distinguish this ID type as being used explicitly by zoos and bioparks.

_**Table**: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type

_**Priority**: high

_**Example Data**: 
Screenshots of these ID types in the ZIMS database:

![Screenshot 2023-06-22 17 25 56](https://github.com/ArctosDB/arctos/assets/14808196/8e05ce29-e95e-4198-bdb0-f1a28edee8e5)

![Screenshot 2023-06-22 17 28 31](https://github.com/ArctosDB/arctos/assets/14808196/162f93b9-f215-42be-8fa9-f6d5ed9f404a)

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This seems fine to me.

**Comment by Jegelewicz** [comment]
I don't think this is necessary until such time that we have two identifiers issued by any single zoo. ABQ BioPark identifiers should be handled as outlined in https://github.com/ArctosDB/arctos/issues/5772

**Comment by dustymc** [comment]
>until such time that we have two identifiers issued by any single zoo.

That would just slightly change the shape of the necessary Agents.

**Comment by campmlc** [comment]
This is my preferred strategy for how to deal with #5772. I did not file #5772 and do not support it. But this option accommodates our data tracking and workflow needs. Also, it would allow us to clean up these Biopark identifiers so that they are in a consistent format and not halfway between multiple different solutions.

**Comment by campmlc** [comment]
Modifying to "Local ID" is also acceptable, but perhaps less clear.

**Comment by Jegelewicz** [comment]
Is this dead? If we could do this and close it as well as https://github.com/ArctosDB/arctos/issues/5772? That would clean up something REALLY old.

**Comment by Jegelewicz** [comment]
After discussion with @campmlc today, re-opening with edits for consideration by the community.

**Comment by ekrimmel** [comment]
I wonder if we could call this "local number" instead of "local ID" just to be consistent with other similar values, like "institutional catalog number," "field number," "collector number" ?

**Comment by Jegelewicz** [comment]
@campmlc thoughts on the comment above?

**Comment by campmlc** [comment]
For our use case, the name of the identifier given by a participating zoo in Species360 is Local ID in exactly that format. I would prefer the more specific term because it clearly is the appropriate value for this context, and I don't see that it would cause confusion for anyone wanting to use it for another purpose. Note that the original value proposed was Zoo Local ID.

**Comment by ekrimmel** [comment]
@campmlc re-reading with more thought to the identifier definition here, I rescind my suggestion. It makes a lot of sense to prioritize aligning this with Species360 over other Arctos identifier types.

**Comment by mkoo** [comment]
Resolved
